src/util/positioning.ts

```diff
@@ -0,0 +1,102 @@
+import {ElementRef} from '@angular/core';
```

Comment:
Doesn't look like we are using `ElementRef` anywhere (and probably shouldn't be) so this import can be removed.